apple/MarkdownTextInputDecoratorShadowNode.mm

```diff
@@ -0,0 +1,255 @@
+#include "MarkdownTextInputDecoratorShadowNode.h"
+
+#include <react/renderer/components/iostextinput/TextInputState.h>
+
+#include <React/RCTUtils.h>
+#include <react/renderer/components/view/conversions.h>
+#include <react/renderer/core/ComponentDescriptor.h>
+#include <react/renderer/textlayoutmanager/RCTAttributedTextUtils.h>
+#include <yoga/Yoga.h>
+
+#include "RCTMarkdownStyle.h"
+#include "RCTMarkdownUtils.h"
+
+namespace facebook {
+namespace react {
+
+extern const char MarkdownTextInputDecoratorViewComponentName[] =
+    "MarkdownTextInputDecoratorView";
+
+void MarkdownTextInputDecoratorShadowNode::initialize() {
+  ShadowNode::traits_.unset(ShadowNodeTraits::ForceFlattenView);
+}
+
+void MarkdownTextInputDecoratorShadowNode::overwriteMeasureContent() {
+  const auto &children = getChildren();
+  if (children.empty()) {
+    return;
+  }
+  react_native_assert(
+      children.size() == 1 &&
+      "MarkdownTextInputDecoratorView received more than one child");
+
+  const auto child = std::dynamic_pointer_cast<const TextInputShadowNode>(children[0]);
+  react_native_assert(
+      child != nullptr &&
+      "MarkdownTextInputDecoratorView received child other than a TextInput");
+
+  // don't mind this :)
+  const auto &nodeWithAccessibleYogaNode =
+      std::reinterpret_pointer_cast<const MarkdownTextInputDecoratorShadowNode>(child);
+  
+  // decorator node cannot have a measure function since it's not a leaf node
+  // but we can redirect measuring of the child input to call measureContent
+  // on the decorator
+  YGNodeSetMeasureFunc(&nodeWithAccessibleYogaNode->yogaNode_,
+                       yogaNodeMeasureCallbackConnector);
```

Comment:
How about this?
```suggestion
  const auto &yogaNode = &nodeWithAccessibleYogaNode->yogaNode_;
  YGNodeSetMeasureFunc(yogaNode, yogaNodeMeasureCallbackConnector);
```

Reply:
[453f718](https://github.com/Expensify/react-native-live-markdown/pull/566/commits/453f7182feeb5b3d2c9a5df3c5a4c07b224fab59)